api/HTMLIFrameElement.json

```diff
@@ -142,66 +142,62 @@
           }
         }
       },
-      "allowFullScreen": {
+      "allowFullscreen": {
         "__compat": {
           "support": {
             "chrome": {
-              "prefix": "webkit",
-              "version_added": "17",
-              "notes": "Daily test builds only."
+              "version_added": "38"
             },
             "chrome_android": {
-              "prefix": "webkit",
-              "version_added": "18",
-              "notes": "Daily test builds only."
+              "version_added": "38"
             },
             "edge": {
-              "version_added": "≤18"
+              "version_added": "≤15"
             },
             "firefox": [
               {
-                "version_added": "18"
+                "version_added": "22"
               },
               {
-                "prefix": "moz",
-                "version_added": "9"
+                "alternative_name": "mozAllowFullScreen",
+                "version_added": "9",
+                "version_removed": "18"
               }
             ],
             "firefox_android": [
               {
-                "version_added": "18"
+                "version_added": "22"
               },
               {
-                "prefix": "moz",
-                "version_added": "9"
+                "alternative_name": "mozAllowFullScreen",
+                "version_added": "9",
+                "version_removed": "18"
               }
             ],
             "ie": {
               "version_added": false
             },
             "opera": {
-              "version_added": false
+              "version_added": "25"
             },
             "opera_android": {
-              "version_added": false
+              "version_added": "25"
             },
             "safari": {
-              "version_added": false
+              "version_added": "10.0"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "10.0"
```

Comment:
```diff
             "safari_ios": {
-              "version_added": null
+              "version_added": "10.0"
```

`10.0` isn’t an allowed version value for iOS Safari — but `10` is, so we need to change this to thate